awx/ui_next/src/components/RoutedTabs/RoutedTabs.jsx

```diff
@@ -57,23 +57,25 @@ function RoutedTabs(props) {
 
   return (
     <Tabs activeKey={getActiveTabId()} onSelect={handleTabSelect}>
-      {tabsArray.map(tab => (
-        <Tab
-          aria-label={`${tab.name}`}
-          eventKey={tab.id}
-          key={tab.id}
-          link={tab.link}
-          title={
-            tab.isNestedTabs ? (
-              <>
-                <CaretLeftIcon /> {tab.name}
-              </>
-            ) : (
-              tab.name
-            )
-          }
-        />
-      ))}
+      {tabsArray
```

Comment:
This block allows use to add a 'Return to ____' tab and render it in in certain nested contexts like Inventory Groups or Inventory Hosts

Reply:
We _could_ move that logic into the component that builds their tab array instead of adding a condition in routed tabs. That would keep the routed tabs component more generic. 

For instance, we could define the basic "Host" tabsArray: [ Details, Facts, Groups, Completed Jobs] in Host.js. If the location doesn't start with `/hosts`, then we could modify the tabsArray to show: [ Back to Hosts, Details, Facts, Groups, Completed Jobs].

Reply:
agreed with @marshmalien.  If we _can_ keep the custom stuff in the calling component, I think that's preferred for now.

I talked with taufique about this one, and he mentioned the "return to..." pattern will only be on specifically nested places like hosts, and not a pattern throughout the app, so I don't think we need to involve the generic component for the handful of instances we will have to do this on.